test/ng/directive/validatorsSpec.js

```diff
@@ -204,6 +204,21 @@ describe('validators', function() {
       expect($rootScope.form.test.$error.pattern).toBe(true);
       expect(inputElm).not.toBeValid();
     });
+
+
+    it('should validate the viewValue and not the modelValue', function() {
+      var inputElm = helper.compileInput('<input type="text" name="test" ng-model="value" pattern="\\d{4}$">');
```

Comment:
I suspect this is a copy-paste from the previous test, but the trailing `$` doesn't seem to be necessary (neither here nor in other tests).